src/hooks/useCookie/useCookie.ts

```diff
@@ -0,0 +1,58 @@
+const useCookie = () => {
+  const getCookie = (name: string) => {
+    let cookieValue = null;
+    if (document.cookie !== '') {
+      const cookies = document.cookie.split(/; */);
+      cookies.forEach((cookie) => {
+        const cookieName = cookie.split('=')[0];
```

Comment:
Could we use destructurization here?

```suggestion
        const [cookieName, cookieVal] = cookie.split('=')
```